source/apiVolontaria/volunteer/tests/tests_admin_Participation.py

```diff
@@ -0,0 +1,111 @@
+        )
+        self.random_state_province = StateProvince.objects.create(
+            name="random state",
+            iso_code="RS",
+            country=self.random_country,
+        )
+        self.address = Address.objects.create(
+            address_line1='random address 1',
+            postal_code='RAN DOM',
+            city='random city',
+            state_province=self.random_state_province,
+            country=self.random_country,
+        )
+        self.cell = Cell.objects.create(
+            name="my cell",
+            address=self.address,
+        )
+        self.cycle = Cycle.objects.create(
+            name="my cycle",
+        )
+        self.task_type = TaskType.objects.create(
+            name="my tasktype",
+        )
+
+        start_date = timezone.now() - timezone.timedelta(
+            minutes=100,
+        )
+        end_date = start_date + timezone.timedelta(
+            minutes=50,
+        )
+
+        self.cycle_inactive = Cycle.objects.create(
+            name="my cycle",
+            start_date=start_date,
+            end_date=end_date,
+        )
+
+        # Some date INSIDE the cycle range
+        start_date = start_date + timezone.timedelta(
+            minutes=1,
+        )
+        end_date = end_date - timezone.timedelta(
+            minutes=1,
+        )
+
+        self.event = Event.objects.create(
+            cell=self.cell,
+            cycle=self.cycle,
+            task_type=self.task_type,
+            start_date=start_date,
+            end_date=end_date,
+        )
+
+        self.partipation = Participation(event=self.event, user=self.user)
+
+    def create_user(self):
+        self.username = "test_admin"
+        self.password = User.objects.make_random_password()
+        user, created = User.objects.get_or_create(username=self.username)
+        user.set_password(self.password)
+        user.is_staff = True
+        user.is_superuser = True
+        user.is_active = True
+        user.save()
+        self.user = user
+
+    def test_admin_access(self):
+        client = Client()
+        client.login(username=self.username, password=self.password)
+
+        admin_pages = [
+            "/admin/volunteer/",
+            "/admin/volunteer/participation/",
+            ]
+
+        for page in admin_pages:
+            resp = client.get(page)
+            self.assertEqual(resp.status_code, 200)
+            self.assertIn("<!DOCTYPE html", str(resp.content))
+
+    def test_admin_participations(self):
+        self.site = AdminSite()
+
+        admin = ParticipationAdmin(Participation, self.site)
+        self.assertNotEqual(admin, None)
+
+        self.assertIsNotNone(admin.user_email(self.partipation))
+        self.assertIsNotNone(admin.user_first_name(self.partipation))
+        self.assertIsNotNone(admin.user_last_name(self.partipation))
+        self.assertIsNotNone(admin.user_phone(self.partipation))
+        self.assertIsNotNone(admin.user_mobile(self.partipation))
```

Comment:
Maybe can you test the value in place of check if it's not None ?
It can be more accurate.